src/figma-default-layers.ts

```diff
@@ -0,0 +1,604 @@
+import * as F from "./figma-json";
+
+// Defaults that non-Figma environments like tests/APIs can use to create figma-json.
+// Update by clicking the plugin's "Log defaults" button and copying the output.
+const defaultLayers: {
+  RECTANGLE: F.RectangleNode;
+  LINE: F.LineNode;
+  ELLIPSE: F.EllipseNode;
+  POLYGON: F.PolygonNode;
+  STAR: F.StarNode;
+  VECTOR: F.VectorNode;
+  TEXT: F.TextNode;
+  FRAME: F.FrameNode;
+} = {
+  RECTANGLE: {
```

Comment:
This might be slightly nicer as individual exports, since you can always grab all of them via `import *`, then just grab what you need righ?

Reply:
Good point, noted.